assets/images/default-image.svg

```diff
@@ -0,0 +1,13 @@
+<svg id="ce515bf4-8650-4a51-8e8f-3fdf36bb1e39" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 800 600">
```

Comment:
Run this through `svgo`, please. There are unnecessary elements and <g>roups, etc.